adapter/tests/test_to_python.py

```diff
@@ -1,29 +1,58 @@
-import unittest
 from unittest import TestCase
 
-from adapter.to_python import Excel, Db
-
+from adapter.to_python import (
+    convert_data_object_to_df,
+    Excel,
+    Db,
+)
+import openpyxl
+import pandas as pd
+from pandas._testing import assert_frame_equal
 import logging
 
 logging.basicConfig(level=logging.DEBUG)
 
 
-class ExcelTests(unittest.TestCase):
+class UtilTest(TestCase):
+    """Tests for general functions"""
+
+    @classmethod
     def setUp(self):
-        """Instantiates an excel loader"""
-        self.exl_loader = Excel('test.xlsx')
+        """Loads an excel workbook"""
+        self.file_path = r"adapter/tests/test.xlsx"
+        self.wb = openpyxl.load_workbook(
+            self.file_path, data_only=True, read_only=False, keep_vba=True
+        )
 
-    def tearDown(self) -> None:
-        del self.exl_loader
+    def test_convert_single_cell_to_df(self):
```

Comment:
These two methods don't use the `Excel` class at all, and they are testing the util method `convert_data_object_to_df`. If we move it into the `ExcelTest` class, it will not be consistent with the `Excel` class.

Reply:
That's right, I am referring to `excel` module and the corresponding test module, rather than the `Excel` class from the `to_python` module.

Reply:
@lyralan Were you able to move the methods? Is it ready for me to I take another look?